common/utilities/tests/test_file_utilities.py

```diff
@@ -0,0 +1,38 @@
+import os
+from unittest import TestCase
+
+from utilities.file_utilities import FileUtilities
+
+TEST_FILE_DIR = "test_files"
+TEST_FILE = "test.txt"
+TEST_STRING = "Test String.\n"
+
+CR_LF_STRING = "foo\r\nbar\r\nbaz"
+CR_STRING = "foo\rbar\rbaz"
+LF_STRING = "foo\nbar\nbaz"
+MIXED_STRING = "foo\r\nbar\nbaz"
+EXPECTED_NORMALIZED_STRING = "foo\nbar\nbaz"
+
+
+class TestFileUtilities(TestCase):
+
+    def test_get_file_string(self):
+        current_dir = os.path.dirname(__file__)
+        test_files_dir = os.path.join(current_dir, TEST_FILE_DIR)
+        test_file = os.path.join(test_files_dir, TEST_FILE)
+
+        loaded_string = FileUtilities.get_file_string(test_file)
+
+        self.assertEqual(TEST_STRING, loaded_string, "The string loaded should match the one expected.")
+
+    def test_normalize_line_endings(self):
```

Comment:
One thing I found quite useful is the `subTest` to group tests together but different variations. Perhaps this could be of use here
`with self.subTest("description of test"):`

Reply:
As discussed offline, this does look like it could be useful to produce a parameterised version of this test. I'll look at doing this in a subsequent change.